samples/createJobFromAdHoc.js

```diff
@@ -26,10 +26,13 @@ function main(projectId, location, inputUri, outputUri) {
   // outputUri = 'gs://my-bucket/my-output-folder/';
 
   // Imports the Transcoder library
-  const {TranscoderServiceClient} = require('@google-cloud/video-transcoder');
+  const {TranscoderServiceClient} =
+    require('@google-cloud/video-transcoder').v1;
 
   // Instantiates a client
-  const transcoderServiceClient = new TranscoderServiceClient();
+  const transcoderServiceClient = new TranscoderServiceClient({
+    version: 'v1',
```

Comment:
I don't believe the `version: 'v1'` here does anything.  Having the `.v1` above should be sufficient.

Reply:
Done